pages/docs/deployment/self-host-v3.mdx

```diff
@@ -0,0 +1,350 @@
+| `DIRECT_URL`                                                                                         | `DATABASE_URL`     | Connection string of your Postgres database used for database migrations. Use this if you want to use a different user for migrations or use connection pooling on `DATABASE_URL`. **For large deployments**, configure the database user with long timeouts as migrations might need a while to complete.                                                                                            |
+| `SHADOW_DATABASE_URL`                                                                                |                    | If your database user lacks the `CREATE DATABASE` permission, you must create a shadow database and configure the "SHADOW_DATABASE_URL". This is often the case if you use a Cloud database. Refer to the [Prisma docs](https://www.prisma.io/docs/orm/prisma-migrate/understanding-prisma-migrate/shadow-database#cloud-hosted-shadow-databases-must-be-created-manually) for detailed instructions. |
+| `CLICKHOUSE_URL`                                                                                | Required                   | Hostname of the clickhouse instance. Pattern: `http(s)://<hostname>:8123` |
+| `CLICKHOUSE_USER`                                                                                | Required                   | Username of the clickhouse database. Needs SELECT, INSERT, CREATE, DELETE grants.   |
+| `CLICKHOUSE_PASSWORD`                                                                                | Required                   | Password of the clickhouse user. |
+| `REDIS_HOST`                                                                                | Required                   | Hostname of the Redis instance.  |
+| `REDIS_PORT`                                                                                | `6379`                   | Port of the Redis instance.  |
+| `REDIS_AUTH`                                                                                | Required                   | Authentication string for the Redis instance.  |
+| `LANGFUSE_S3_EVENT_UPLOAD_ENABLED`<br/>`LANGFUSE_S3_EVENT_UPLOAD_BUCKET`<br/>`LANGFUSE_S3_EVENT_UPLOAD_REGION`<br/>`LANGFUSE_S3_EVENT_UPLOAD_ACCESS_KEY_ID`<br/>`LANGFUSE_S3_EVENT_UPLOAD_SECRET_ACCESS_KEY`<br/>`LANGFUSE_S3_EVENT_UPLOAD_ENDPOINT`<br/>`LANGFUSE_S3_EVENT_UPLOAD_FORCE_PATH_STYLE` | Required                   | S3 configuration to upload incoming events into S3. `LANGFUSE_S3_EVENT_UPLOAD_BUCKET` is required for Langfuse to accept events. The other variables are optional and will use the default provider credential chain if not specified.                                                                                                                                                                               |
```

Comment:
I'll add a note that this is mainly relevant for AWS instances. There are certain cases where it would apply on other cloud vendors, but it would be very uninuitive and an uncommon approach.